packages/edit-post/src/components/header/template-title/template-description.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { __experimentalText as Text } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { store as editPostStore } from '../../../store';
+
+export default function TemplateDescription() {
+	const { description } = useSelect( ( select ) => {
+		const { getEditedPostTemplate } = select( editPostStore );
+		return {
+			description: getEditedPostTemplate().description,
+		};
+	}, [] );
+	return !! description && <Text size="body">{ description }</Text>;
```

Comment:
Just nitpicking, but maybe we should explicitly return `null` if there's no description. A similar expression without `!!` will result in an error if there's no description.